broker/src/main/java/org/apache/rocketmq/broker/processor/ReplyMessageProcessor.java

```diff
@@ -263,7 +263,7 @@ private void handlePutMessageResult(PutMessageResult putMessageResult,
                 break;
             case SERVICE_NOT_AVAILABLE:
                 log.info(
-                    "service not available now, maybe disk full, maybe your broker machine memory too small.");
+                    "service not available now, maybe disk full, maybe put messages to slave, maybe message store shutdown, etc.");
```

Comment:
Too many uncertain reasons can drive people crazy. You are advised to change them to “you could check...” if not any diagnosis could be provided furtherly.